src/Components/PairingForm.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { findGame } from '../redux/actions/boardGame';
+import { setLiquorType, setRandomDrink } from '../redux/actions/liquorSearch'
+
+const PairingForm = () => {
+
+    const dispatch = useDispatch()
+
+    const games = useSelector(state => state.boardGame.allGamesData.games)
+    const type = useSelector(state => state.liquorSearch.liquorSearchWord)
+
+    const gameNames = games.map((game) => {
+        return (
+            <option key={game.id} value={game.name} />
+        )
+    })
+
+    const findRandomDrink = (event) => {
+        event.preventDefault()
+        fetch(`https://www.thecocktaildb.com/api/json/v1/1/filter.php?i=${type}`)
+            .then(res => res.json())
+                .then(data => {
+                    console.log('data', data)
+                    let randomNum = Math.floor(Math.random() * data.drinks.length)
+                    console.log("dataaRando", randomNum)
+                    dispatch(setRandomDrink(data.drinks[randomNum]))
+                })
+    }
+
+    return (    
+        <form>
+            <label htmlFor='game-choice'>
+                Choose Your Game!
+                <input id='game-choice' list='games' onChange={(event) => dispatch(findGame(event.target.value))}/>
+                    <datalist id='games'>
```

Comment:
Love the `datalist` input type @michelecomfort - thanks for sharing this with us.